### ISSUE TITLE
Review client type distribution collection

### ISSUE THREAD

**Issue by philknows** [ChainSafe/eth2-crawler#208]
## Expected Behavior
Currently the crawler is pulling a count for the `Client type distribution` metric which unfortunately does not look accurate in distribution. Other crawlers such as clientdiversity.org is reporting very different distributions using [Miga Labs Armiarma](https://github.com/migalabs/armiarma) and [Sigma Prime's Blockprint](https://github.com/sigp/blockprint) crawlers.

Might be related to #205 

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Versions
Eth2-Crawler commit (or docker tag):
Go version:

**Comment by spacesailor24** [comment]
From my understanding, we're asking each node to report it's `AgentName` (code [here](https://github.com/ChainSafe/nodewatch-api/blob/55bf484780fddaa3154c9b77d43833dcbdd0eee2/crawler/p2p/host.go#L98)) like Miga Labs' crawler, so our numbers should probably be closer to theirs. I think it would make sense that the lack of a digest filter (#205) would be causing our skewed numbers. I think the best way to test this would be to connect an API with the fork digest filter to our prod DB and apply corresponding mainnet fork digests and see if the numbers are closer to what Miga Labs is reporting

```
┌─────────┬──────────────┬──────────────┬──────────────┐
│ (index) │    phase0    │    altair    │  bellatrix   │
├─────────┼──────────────┼──────────────┼──────────────┤
│ mainnet │ '0xb5303f2a' │ '0xafcaaba0' │ '0x4a26c58b' │
│ goerli  │ '0x79df0428' │ '0x82f4a72b' │ '0xc2ce3aa8' │
│ ropsten │ '0x4be89531' │ '0xf56931b6' │ '0x3cfa3bac' │
│ sepolia │ '0xa8fee8ee' │ '0x60725ca1' │ '0x36fa5013' │
└─────────┴──────────────┴──────────────┴──────────────┘
```

**Comment by sadiq1971** [comment]
> 

Right. A test with the updated api on prod will give us more accurate result. I am also waiting for testing.
